package.json

```diff
@@ -29,12 +29,13 @@
   "homepage": "https://github.com/stampit-org/stamp-specification#readme",
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-eslint": "^5.0.0",
+    "babel-eslint": "^6.1.2",
     "babel-polyfill": "^6.7.2",
     "babel-preset-es2015": "^6.6.0",
     "babel-register": "^6.7.2",
-    "check-compose": "^2.0.0",
-    "eslint": "~2.2.0"
+    "check-compose": "^3.1.2",
+    "eslint": "^2.13.1",
```

Comment:
If I understand correctly, you don't want to go with ESlint v3 anytime soon since it has dropped support for Node < 4. I am wondering when it will be right time to drop that support from stampit as well. I think that people who deliberately stay on 0.12 are kinda kicking to their own knees as there is a lot of performance improvements missing not even mentioning native support for a lot of ES6 stuff.